packages/astro/src/@types/astro.ts

```diff
@@ -2112,6 +2112,11 @@ interface AstroSharedContext<
 	 */
 
 	preferredLocaleList: string[] | undefined;
+
+	/**
+	 * The current locale computed from the URL of the request. It matches the locales in `i18n.locales`, and returns `undefined` otherwise.
+	 */
+	currentLocale: string | undefined;
```

Comment:
Just checking that there will eventually be a corresponding docs entry in this file for this, too, like the other ones currently listed in configuration reference?

Reply:
I was hoping that this would be auto-generated, but it's not. I am not **when** we should update this https://docs.astro.build/en/reference/api-reference/#astro-global, but I suppose we should do it right before going out of experimental? 

Please let me know, and I will create a task for me to write the documentation

Reply:
Ah, great point! 

To answer your question, No, these don't get auto-generated.

But, you made me realize that yes, this is not a **configuration** setting, and yes, should go on the `api-reference` instead, so no `@docs` are actually needed here in this file.

And yes, I agree that we'll keep it OFF the api reference page while it's still experimental, so nothing else required. That was my mistake!